internal/output_workflow/output_workflow.go

```diff
@@ -0,0 +1,145 @@
+package output_workflow
```

Comment:
This is as far as I've gotten with my first pass review. Leaving myself a bookmark :)